src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import "../styles/header";
+@import "../styles/card";
+@import "../styles/stars";
+@import "../styles/reset";
```

Comment:
1. acording to check list we should create function - p.13.
2.  where is some problem with TEST REPORT LINK - tests dont display on screen. Maybe because you add 'data-qa="card-hover" to each of cards.  (add only to first card)